maven-enforcer-rules/README.md

```diff
@@ -33,11 +33,11 @@ Here is an example pom.xml:
                             <helidonDependenciesRule>
                                 <namespace>JAKARTA | JAVAX</namespace>
                                  <!-- list of strings - can be used to exclude a package / group name from validation -->
-                                 <excludedGavRegExs>
-                                    <!-- for example only - we suggest not including this exclusion in your usage -->
-                                    <excludedGavRegEx>javax.servlet.*</excludedGavRegEx>
+<!--                                 <excludedGavRegExs>-->
+<!--                                    &lt;!&ndash; for example only - we suggest not including this exclusion in your usage &ndash;&gt;-->
+<!--                                    <excludedGavRegEx>javax.servlet.*</excludedGavRegEx>-->
                                  </excludedGavs>
```

Comment:
Should this be `</excludedGavRegExs>` ?  And the opening element is commented out and this one is not. And why the html encoding that results in `–>-->`

To me the commenting out looks a bit messy. And if we don't want people to exclude servlet then maybe we shouldn't show it at all. Maybe better to not comment out and have `</excludedGavRegEx>`